redhat-cop/casl-ansible
# Updated inventory for demo purpose

#### What does this PR do?
Showing the use of a `content_dir` approach

#### How should this be manually tested?
Use the tests in the `openshift-applier` role with the sample inventory and observe that the files in the directory gets sourced.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl
